### PR TITLE
SEO web page tags recipe

### PR DESCRIPTION
**Pull request by summerknight**

### CODE REVIEW

**Comment by summerknight** [comment]
Btw, Alexander, is it ok to write recipe about how to enable pretty URL? I know it is already mentioned in a lot of places. But maybe it will be good to have it inside SEO essentials section. In order to have all such information in one place.

File: book/web-page-seo-tags.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+Web page SEO tags
+=======================================
+
+Organic extradition is an excellent free traffic source. But in order to have such kind of traffic you have
```
Comment (samdark) [inline book/web-page-seo-tags.md]: extradition?

Comment (samdark) [inline book/web-page-seo-tags.md]: You mean organic search?

Comment (summerknight) [inline book/web-page-seo-tags.md]: Sure, sorry.

**Comment by samdark** [comment]
@summerknight yes, that's a good idea.

**Comment by samdark** [comment]
Another topics could be canonical URLs and slugs.

**Comment by summerknight** [comment]
Another question related to SEO theme. GridView and it's filters. By default when you use filters a very long query string is appeared. Is there a good explanation how to make it pretty?

For SEO it may be ok, for example if you want to increase amount of pages based on GridView filters (of course have to manage possible content duplications because of this).
In other situations for SEO PJAX will be ok.

**Comment by samdark** [comment]
A long query string is actually good for SEO in this case. You definitely don't want all these variations in your search index.

**Comment by summerknight** [comment]
Is it possible to change GridView search link format? In order to avoid something like this (GridView by default):
specialties?Search%5Bname%5D=&Search%5Baddr%5D=fd&Search%5Bphone%5D=&Search%5Bsite%5D=

Sometimes it is not good, for example if user wants to share the link. It is very long and very strange so users will not click on it.

**Comment by samdark** [comment]
Probably yes. I haven't tried to.

**Comment by summerknight** [comment]
Two new recipes are added: how to leverage browser caching, reusing views via partials

**Comment by samdark** [comment]
:+1:

**Comment by samdark** [comment]
Merged with adjustments. Thanks!

**Pull request closed without merging**